src/library/scala/math/package.scala

```diff
@@ -58,7 +58,21 @@ package object math {
    *          logarithms.
    */
   def exp(x: Double): Double = java.lang.Math.exp(x)
+
+  /** Returns the natural logarithm of a `double` value.
+   * 
+   * The natural logarithm is the logarithm base Euler's number `e`.
+   *
+   *  @param  x the number to take the natural logarithm of
+   *  @return the value `ln(x)` where ln is the natural lograrithm
```

Comment:
Maybe say "the value `y` such that e^y^ == `x`, where `e` is Euler's number", or somesuch.